package.json

```diff
@@ -10,8 +10,8 @@
     "lint": "vue-cli-service lint",
     "lint:style": "vue-cli-service lint:style",
     "release": "git checkout develop & git pull & git push origin develop:release",
-    "release:android": "git checkout develop && node ./tasks/bump-android-version.js && git commit android/app/build.gradle -m 'updated android app version' && git push origin develop:release-android",
-    "release:ios": "git checkout develop && node ./tasks/bump-ios-version.js && git commit ios/App/App/Info.plist -m 'updated ios app version' && git push origin develop:release-ios",
+    "release:android": "git checkout develop & git pull & git push origin develop:release-android",
```

Comment:
@jbibla I think we should remove this completely. When there is a fix for mobile we just create a new release.

Reply:
sure! whatever you think is best!